core/modules/system/system.menu.inc

```diff
@@ -109,6 +110,17 @@ function system_menu_block_form($config) {
     '#default_value' => $config['toggle'],
     '#description' => t('A menu toggle button, commonly known as a hamburger icon, allows the menu to be hidden initially on devices with small screens, but then easily expanded.'),
   );
+  $form['expand_all'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Expand all'),
+    '#default_value' => $config['expand_all'],
+    '#description' => t('Show every menu children, not only active trail.'),
```

Comment:
Can we change this text to `When checked, all menu links will be displayed. When unchecked, only menu links in the active trail will be displayed.` This text explains in more detail what happens both when checked and unchecked.